en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/analytics-configurations.md

```diff
@@ -0,0 +1,45 @@
+# Analytics Configurations in config.toml
+The following are the configurations with regard to Analytics. The configuration file ( `config.toml` ) for the Choreo Connect is located in the `CHOREO-CONNECT_HOME/docker-compose/choreo-connect/conf` directory.
+
+To learn about the concepts of Analytics in Choreo Connect, please follow the [Choreo Connect Analytics]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics/)
+
+|Heading|Description|
+|---------------------------------------|-----------------------------|
+|`enabled`  | The configurations required to either enable or disable Analytics Event publishing to Choreo and is used by both `enforcer` and `router`|
+|`analytics.adapter`| The configurations that would be applied into the `router`'s [gRPC Access Logger]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics/#grpc-access-logger). The events will be published to the [gRPC Event Listener]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics/#grpc-event-listener) once `bufferFlushInterval` passes or buffer size exceeds the `bufferSizeBytes`. `gRPCRequestTimeout` is the timeout for the gRPC call from [gRPC Access Logger]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/choreo-connect-analytics/#grpc-access-logger).|
```

Comment:
`gRPCRequestTimeout` is the **Connect** timeout

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/3824/commits/65e640c0328372d499b3747510cb8d7ef954ff4e